### DISCUSSION TITLE
Fully repaired submarine walls sometimes appear as slightly damaged if there are other damaged walls nearby

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#15358] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Occasionally in multiplayer I see small cracks on fully repaired walls. Those are seemingly caused by the presence of broken walls nearby, as after welding the nearby walls, the fake cracks disappear.

### Reproduction steps

1. Play a round of campaign in multiplayer (?)
2. Allow a large portion of the submarine to get damaged, having a bunch of broken walls next to each other, especially where vertical walls meet horizontal ones.
3. Weld one of the walls fully, while leaving others in a damaged state.
4. Sometimes fake cracks will remain on a wall you already welded.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by jigardi** [comment]
I've experienced this one a lot lately, and I don't like it at all.

It makes me distrust/second guess the status monitor constantly,  and it wastes everyone's time by luring people on a fruitless goose-chase outside the sub.

During my last session, it eventually lead to the entire crew completely ignoring external sub damage.

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/6675 (PR in out private repo), fix coming up in the next unstable build and an upcoming release.
